### PR TITLE
Added Markdown formatting to examples/mnist_siamese.py

### PR DESCRIPTION
**Pull request by joosephook**
### Summary
Added Markdown formatting to `examples/mnist_siamese.py`.
Result:
mnist_siamese1
mnist_siamese2

### Related Issues
#12219 
### PR Overview

- [ ] This PR requires new unit tests [y/n] (make sure tests are included)
- [ ] This PR requires to update the documentation [y/n] (make sure the docs are up-to-date)
- [x] This PR is backwards compatible [y/n]
- [ ] This PR changes the current API [y/n] (all API changes need to be approved by fchollet)

### CODE REVIEW

**Comment by taehoonlee** [comment]
Thank you for the PR, @joosephook. Are the #12430-#12436 included in this PR, right? If so, please close the other PRs.

**Comment by joosephook** [comment]
I will close the others. I don't understand how to properly do multiple pull requests so that they would be truly independent - for some reason, the previous commits are included.

**Comment by joosephook** [comment]
Closed the other PRs.

File: examples/mnist_cnn.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
-'''Trains a simple convnet on the MNIST dataset.
+'''
+#Trains a simple convnet on the MNIST dataset.
```
Comment (fchollet) [inline examples/mnist_cnn.py]: Add a space after `#` (applies to all modifications in this PR).

**Comment by fchollet** [comment]
Thanks for the PR.

We are postponing the addition of new examples on keras.io pending a future refactor of the docs website (which will have a dedicated examples section).

**Pull request closed without merging**